src/Components/AddTask/AddTask.tsx

```diff
@@ -1,67 +1,114 @@
 
 import axios from "axios";
 import { useContext, useEffect, useState } from "react";
-import { useNavigate } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import { AuthContext } from "./../../Context/AuthContext";
 import { ToastContext } from "../../Context/ToastContext";
 import { SubmitHandler, useForm } from "react-hook-form";
 import { ProjectContext } from '../../Context/ProjectContext';
-import { FormValues, projectType ,IAuth} from './../../Types/Types';
+import { FormValues, projectType, IAuth } from './../../Types/Types';
 
 
 const AddTask: React.FC = () => {
-  const { baseUrl, requestHeaders }:any
-  //Pick<IAuth, 'baseUrl','requestHeaders'> 
-  = useContext(AuthContext);
+  const { baseUrl, requestHeaders }: any
+    //Pick<IAuth, 'baseUrl','requestHeaders'>
+    = useContext(AuthContext);
   const { getToastValue }: void = useContext(ToastContext);
-  const {  projects }: projectType[] = useContext(ProjectContext);
+  // const { projects }: projectType[] = useContext(ProjectContext);
   // console.log(projects, "from addtasks");
   const [isLoading, setIsLoading] = useState(false);
   const navigate = useNavigate();
+  const [userList, setUserList] = useState([])
+  const [projectList,setProjectList]=useState([])
+
 
-  
   const {
     register,
     handleSubmit,
     formState: { errors },
   } = useForm<FormValues>();
 
-  const goBack=()=>{
+  const goBack = () => {
     navigate("/dashboard/tasks");
```

Comment:
Using navigate(-1) will navigate to the previous page, so you don't need to write a complete function for that.

![image](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/1a6bf252-5e4d-4f30-81d0-c85dfbf936e0)